.pre-commit-config.yaml

```diff
@@ -44,7 +44,7 @@ repos:
       - id: docformatter
         args: ["--in-place", "--wrap-descriptions", "79"]
   - repo: https://github.com/open-mmlab/pre-commit-hooks
-    rev: master  # Use the ref you want to point at
+    rev: v0.1.0  # Use the ref you want to point at
```

Comment:
Why to update pre-commit?

Reply:
The master branch was updated two weeks ago. Pre-commit can't be passed by using the latest master branch.

Reply:
ok